.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -63,9 +63,9 @@ This change added tests and can be verified as follows:
   - The serializers: (yes / no / don't know)
   - The runtime per-record code paths (performance sensitive): (yes / no / don't know)
   - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (yes / no / don't know)
+  - The S3 file system connector: (yes / no / don't know)
```

Comment:
Maybe we could add some more details for the committer why this is important. E.g. pointing to the documentation where we explain what's the problem with it.

Reply:
Yes definitely. Also maybe with some small manual how to run those S3 tests?

Reply:
@pnowojski The S3 tests are automatically run if the S3 credentials are correctly setup on travis.

Reply:
Wouldn't this be true for any of the points we ask users to fill out? In the end, there should probably be a committer's guide (is there?) with more details on everything but for the committer only - we don't want to scare the contributer away.